mxfusion/runtime/distributions/multivariate_normal.py

```diff
@@ -149,3 +150,14 @@ def kl_divergence(self, other):
         return -M/2 + sumlogdiag(L_2) - sumlogdiag(L_1).sum() + \
             mx.nd.square(LinvLs).sum(-1).sum(-1)/2 + \
             mx.nd.square(Linvmu).sum(-1).sum(-1)/2
+
+    def broadcast_to_n_samples(self, n_samples):
```

Comment:
We have help functions in mxfusion.components.variables.runtime_variables. Can we avoid this function?